find_max.py

```diff
@@ -0,0 +1,16 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index):
+    max = None
+    if max_index == 0:
```

Comment:
is the max_index last element in the array? if so, is it necessary to have it as an argument? we could instead find the last element index and whether or not the list is empty by using len(array). however, if the max_index is just the index in which you want to check until, I'm not sure if it being == 0 would necessarily mean that the list was empty? we might want to then change the comment to 'max index must not be 0' or something like that.